src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/HotClusterSummary.java

```diff
@@ -57,6 +57,7 @@ public HotClusterSummary(int numOfNodes, int numOfUnhealthyNodes) {
     super();
     this.numOfNodes = numOfNodes;
     this.numOfUnhealthyNodes = numOfUnhealthyNodes;
+    this.hotNodeSummaryList = new ArrayList<>();
```

Comment:
Did this cause a bug somewhere? Is this a necessary change?

Reply:
yes, caused an NPE without it.